website/blog/2021-05-25-inking.md

```diff
@@ -0,0 +1,409 @@
+---
+title: "Add inking to your React Native for Windows application"
+author: Matteo Pagani
+authorURL: https://twitter.com/qmatteoq
+description: "How to use add inking to a React Native for Windows application"
+is_blog: true
+published_date: May 25, 2021
+---
+
+Windows 10 is one of the most powerful platforms on the market when it comes to supporting digital inking experiences. As such, the Universal Windows Platform comes with a dedicated series of controls that you can add to your application to enable inking experiences. Thanks to these controls, you can easily implements features like capturing signatures, text and shape recognition, etc.
+
+If you're building a Windows application with React Native, there are a few options you can explore to add such a feature. One of them is using a community module called [react-native-sketch-canvas](https://github.com/creambyemute/react-native-sketch-canvas), which provides a Windows implementation as well. This module is based on Direct2D and it enables the recording and playback of ink strokes. While this solution can be a great fit for many applications, there are scenarios in which you need more precision and better integration with the Windows ecosystem, like when you need to capture a signature. For these purposes, Windows 10 offers a native control called [`InkCanvas`](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.InkCanvas),  which enables powerful inking experiences. Since `InkCanvas` is a specific Windows 10 control, it isn't automatically exposed by React Native, since we don't have an equivalent control on other platforms. As such, we'll have to wrap it with a [native UI component](https://microsoft.github.io/react-native-windows/docs/view-managers), which is a feature provided by React Native which exposes native UI controls for Windows, Android or iOS to the JavaScript layer using JSX.
```

Comment:
```suggestion
If you're building a Windows application with React Native, there are a few options you can explore to add such a feature. One of them is using a community module called [react-native-sketch-canvas](https://github.com/creambyemute/react-native-sketch-canvas), which provides a Windows implementation as well. This module is based on Win2D and it enables the recording and playback of ink strokes. While this solution can be a great fit for many applications, there are scenarios in which you need more precision and better integration with the Windows ecosystem, like when you need to capture a signature. For these purposes, Windows 10 offers a native control called [`InkCanvas`](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.InkCanvas),  which enables powerful inking experiences. Since `InkCanvas` is a specific Windows 10 control, it isn't automatically exposed by React Native, since we don't have an equivalent control on other platforms. As such, we'll have to wrap it with a [native UI component](https://microsoft.github.io/react-native-windows/docs/view-managers), which is a feature provided by React Native which exposes native UI controls for Windows, Android or iOS to the JavaScript layer using JSX.
```